_pages/business-units/18f/projects-partners/collaboration-tools.md

```diff
@@ -172,6 +172,24 @@ Here are some common barriers to using Google Calendar and some workarounds to t
 - Truly low-impact: Mural shouldn't be used to store anything confidential or authoritative. Instead, use it for ephemeral organization of information and production visualizations. Non-GSA participants are invited to use the service anonymously.
 </details>
 
+### Feedback Collection
+
+**Qualtrics** is a feedback collection tool that helps with creating surveys. These surveys can be distributed in a variety of ways including on websites like [usa.gov](https://www.usa.gov/); and used to create data dashboards. Any TTS employee may use this tool for free. Licenses may be aquired by contacting GSA's Office of Customer Experience (OCE) at customerexperience@gsa.gov. Please make sure to state your business case in your request. 
+
+- [Qualtrics handbook page]
```

Comment:
Let's remove, since there isn't a dedicated page for it.

```suggestion
```

Reply:
> A few suggested edits. @sheevdave Is there an InSite page or Google Doc or somewhere canonical we can link to? This information is good, and seems applicable to all of GSA.

https://insite.gsa.gov/services-and-offices/staff-offices/office-of-customer-experience/surveys-and-feedback-collection

Reply:
Sounds good let's delete it

Reply:
Also heads up, I'm planning to create a dedicated page for it eventually.